models/POP/shell_scripts/cesm1_x/CESM1_1_1_setup_hybrid

```diff
@@ -747,9 +747,9 @@ cat << EndOfText >! stage_cesm_files
 set restart_time = $init_time
 
 # get the settings for this case from the CESM environment
-cd ${caseroot}
+cd \${caseroot}
```

Comment:
I am not sure this one needs 'escaping' ... it will expand into the full value when the script is being created, which is what I think should happen.   The variables that it gets from the ccsm_getenv call must be 'escaped', agreed.